spec/factories/legal_aid_applications.rb

```diff
@@ -69,6 +71,43 @@
       end
     end
 
+    # The trait :with_scope_limitations will set up:
+    #
+    # * one proceeding type
+    # * a default substantive scope limitation for that proceeding type
+    # * a default delegated functions scope limitation for that proceeding type
+    # * add the default substantive scope limitation to the application
+    # * add the default delegated functions scope limitation to the application (SEE NOTE BELOW)
+    #
+    # NOTE: The delegated function scope limitation will only be added to the application if
+    # the :with_delegated_functions trait is specified BEFORE the :with_scope_limitations trait
+    # when created, e.g.
+    #
+    #     create :legal_aid_application, :with_delegated_functions, :with_scope_limitations
+    # will work.
+    #
+    #    create :legal_aid_application, :with_scope_limitations, :with_delegated_functions
+    # will NOT work
+    #
+    # See factory specs at spec/factory_specs/legal_aid_application_factory_spec.rb
+    #
+    trait :with_scope_limitations do
+      after(:create) do |application, evaluator|
+        proceeding_types = if evaluator.proceeding_types.empty?
+                             create_list(:proceeding_type, 1)
```

Comment:
If this is changed to:
```ruby
create_list(:proceeding_type, 1, legal_aid_application: application)
```
line 101 becomes redundant, and line 103 can be simplified to:
```ruby
proceeding_type = proceeding_types.first
```